kartotherian/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN git clone https://github.com/QwantResearch/kartotherian.git /opt/kartotheria
     && npm install --production \
     && git clone https://github.com/QwantResearch/kartotherian_config.git /opt/kartotherian_config \
     && cd /opt/kartotherian_config \
-    && git checkout 78208ef48b925c1e81fdad4dd47227b3d31f7fe1
+    && git checkout 6bc774c19bd236ca6a362478bfcb556a69455d97
```

Comment:
please update this commit to use the latest master now that https://github.com/QwantResearch/kartotherian_config/pull/48 is merged